src/main/resources/jenkins/plugins/office365connector/WebhookJobProperty/config.jelly

```diff
@@ -3,75 +3,98 @@
-                                    </f:entry>
-                                </td>
-                            </tr>
-                            <tr>
-                            	<td>
-                            		<f:entry title="Notify Build Start">
-      								<f:checkbox name="startNotification" value="true" checked="${webhook.isStartNotification()}"/>
-    								</f:entry>
-
-        							<f:entry title="Notify Aborted">
-            							<f:checkbox name="notifyAborted" value="true" checked="${webhook.isNotifyAborted()}"/>
-        							</f:entry>
-
-        							<f:entry title="Notify Failure">
-            							<f:checkbox name="notifyFailure" value="true" checked="${webhook.isNotifyFailure()}"/>
-        							</f:entry>
-
-        							<f:entry title="Notify Not Built">
-            							<f:checkbox name="notifyNotBuilt" value="true" checked="${webhook.isNotifyNotBuilt()}"/>
-        							</f:entry>
-
-        							<f:entry title="Notify Success">
-            							<f:checkbox name="notifySuccess" value="true" checked="${webhook.isNotifySuccess()}"/>
-        							</f:entry>
-
-        							<f:entry title="Notify Unstable">
-            							<f:checkbox name="notifyUnstable" value="true" checked="${webhook.isNotifyUnstable()}"/>
-        							</f:entry>
-
-        							<f:entry title="Notify Back To Normal">
-            							<f:checkbox name="notifyBackToNormal" value="true" checked="${webhook.isNotifyBackToNormal()}"/>
-        							</f:entry>
-                                                                
-                                                                <f:entry title="Notify Repeated Failure">
-            							<f:checkbox name="notifyRepeatedFailure" value="true" checked="${webhook.isNotifyRepeatedFailure()}"/>
-        							</f:entry>
-                            	</td>
-                            </tr>
-                            <tr>
-                                <td>
-                                    <f:entry title="Timeout" description="Timeout (in ms)"
-                                        field="timeout">
-                                        <f:textbox name="timeout" value="${webhook.getTimeout()}" default="${descriptor.defaultTimeout}"/>
-                                    </f:entry>
-                                </td>
-                            </tr>
-                        </table>
-                    </f:entry>
-                    <f:repeatableDeleteButton value="${%Delete}" />
-                </table>
-            </f:repeatable>
-        </f:entry>
-    </f:section>
+  				<table>
+					<f:entry field="webhook">
+
+						<f:entry title="Name" field="name">
+							<f:textbox name="name" value="${webhook.getName()}" />
+						</f:entry>
+
+						<f:entry title="URL" field="url">
+							<f:textbox name="url" value="${webhook.getUrl()}" />
+						</f:entry>
+
+						<f:advanced>
```

Comment:
Here we have advanced section which is by default collapsed (and user can expand it when need to apply modification). As this section becomes longer and longer (and I have plan to extend it in near future) it seems to be more user friendly that having everything expanded and scroll it all the time